backend/app/processing/llm.py

```
@@ -0,0 +1,369 @@
+        else:
+            reduced_prompt.append(f"## {section}")
+            reduced_prompt.extend(lines)
+    
+    return "\n".join(reduced_prompt)
+
+def process_query_llm(chatbot_id: str, query: str, context_docs: list, conversation_history=None) -> str:
+    """Process a query using the appropriate context and LLM"""
+    prompt = create_prompt(query, context_docs, conversation_history)
+    logger.info(f"Sending prompt to Groq LLM for chatbot {chatbot_id}")
+    
+    estimated_tokens = estimate_token_count(prompt)
```

Ruff (0.8.2)

Local variable `estimated_tokens` is assigned to but never used

Remove assignment to unused variable `estimated_tokens`

(F841)